TheJumpCloud/support
# Added fix to string value error

## Issues
* [SA-3679](https://jumpcloud.atlassian.net/browse/SA-3679) - SA-3679-PolicytoPatch-Copy

## What does this solve?
Fixes issue with policy value converting a string to an integer error
## Is there anything particularly tricky?
n/a
## How should this be tested?
1. Run this with the current module: Set-JCPolicy -id POLICYID -AUTO_INSTALL_SCHEDULE ScheduledInstallFourthWeek
* This should output an error
2. Run the same code with this module, validate that the issue is fixed
## Screenshots


[SA-3679]: https://jumpcloud.atlassian.net/browse/SA-3679?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
